app/Filament/Tenant/Resources/ProductResource/Traits/HasProductForm.php

```diff
@@ -177,4 +178,27 @@ public function generateExpiredFormComponent()
             ->required()
             ->native(false);
     }
+
+    public function generateAdditionalBarcodesFormComponent(): Repeater
```

Comment:
Get behavior error, the product and the additional barcode can be created on the database, but the form wasn't redirected to the index page because of getting error The additional barcode has already been taken.|